Makefile

```diff
@@ -201,6 +202,38 @@ download_or_build_containers:
 # Update databases
 # ==============================================================================
 
+# -----------------------------
+# Download Hostile human index
+# -----------------------------
+
+update_hostile_db: download_minimap2_index_genome \
+	download_bowtie2_genome_index
+
+HOSTILE_DIR := $(ASSETS_DIR)/hostile_db
+download_minimap2_index_genome: $(HOSTILE_DIR)/human-t2t-hla.fa.gz
+
+$(HOSTILE_DIR)/human-t2t-hla.fa.gz:
+	$(call log_message,"Starting download of Hostile human index")
+	mkdir -p $(HOSTILE_DIR) \
+	&& cd $(HOSTILE_DIR) \
+	&& wget https://objectstorage.uk-london-1.oraclecloud.com/n/lrbvkel2wjot/b/human-genome-bucket/o/human-t2t-hla.fa.gz
```

Comment:
Is there a missing trailing `\` here?

Reply:
Yeah corrected it when I saw the run failed!